src/impact-dashboard/FacilityInformation.tsx

```diff
@@ -172,6 +172,24 @@ interface AgeGroupRowProps {
 
 const AgeGroupRow: React.FC<AgeGroupRowProps> = (props) => {
   const { model, updateModel } = props;
+  const [inputRelativityError, setInputRelativityError] = useState(false);
+
+  function checkInputRelativity(
+    cases: number | undefined,
+    total: number | undefined,
+  ) {
+    if (cases == undefined) {
+      return setInputRelativityError(false);
```

Comment:
Are these return statements just being used to terminate at the right location? I have not seen that convention before, personally! I'd probably opt to just have this by one cohesive conditional block:
```
if (cases == undefined) {
  setInputRelativityError(false);
} else if (cases != undefined && total === undefined) {
  setInputRelativityError(true);
} else if (total != undefined && cases > total) {
  setInputRelativityError(true);
} else {
  setInputRelativityError(false);
}
```

Reply:
As a general rule, I'll defer to others on JavaScript. So I'll fix this up this afternoon.

Reply:
FWIW, this is definitely a convention, nothing wrong with it, just a matter of style. arguably having a big conditional block is a more readable signal that we are checking a series of mutually exclusive conditions but six of one IMO